.github/workflows/rust.yml

```diff
@@ -25,7 +25,11 @@ jobs:
           key: "${{runner.os}} Rust ${{steps.rust-toolchain.outputs.cachekey}} Lock ${{hashFiles('Cargo.lock')}}"
       - uses: msrd0/install-rlottie-action@v1
       - run: sudo apt-get update -y && sudo apt-get install -y $PACKAGES
-      - run: cargo test --workspace --all-features --release -- --include-ignored
+      - name: install cargo-all-features
+        uses: baptiste0928/cargo-install@v1.3.0
+        with:
+          crate: cargo-all-features
+      - run: cargo test-all-features --jobs 1 --release -- --include-ignored
```

Comment:
You probably don't want to reduce the amount of jobs used for compilation, but rather for the tests executed.